### ISSUE TITLE
Scaling and crop with mpv backend

### ISSUE THREAD

**Issue by no95typem** [catsout/wallpaper-engine-kde-plugin#17]
**Describe the bug**
"Scaling and crop" display option doesn't work with mpv backend.

**Comment by catsout** [comment]
Sadly, I don't find a corresponding option for this in mpv.

**Comment by no95typem** [comment]
This issue doesn't matter as it turns out. "Scale to fill" with mpv is equal to "Scaling and crop" without it. So I think that now "Scale to fill"  option with mpv actually is "Scaling and crop". Maybe one of "if" statements should be changed in mpv.qml #18 ? And as for "Scale to fill" functionality, I think this display mode doesn't look very nice, but it can be implemented with (video-scale-x) mpv property.

**Comment by catsout** [comment]
There is a `fillmode` in qt  
https://doc.qt.io/qt-5/qml-qtquick-image.html#fillMode-prop  

The "Scaling and crop" is equal to "PreserveAspectCrop"  
"Scale to fill" is equal to "Stretch"

**Comment by no95typem** [comment]
Yes, but it's about QtMultimedia backend. But I talk about mpv backend and panscan = 1.0. I think "panscan = 1.0" option is equal to "PreserveAspectCrop" in qt. I made such conclusions from the description of this option [here.](https://github.com/mpv-player/mpv/blob/master/DOCS/man/options.rst). And  I tried mpv backend in the mode "Scale to fill", and then switch immediately to QtMultimedia backend with the mode "Scaling and crop". And I can't see the difference on a 2560x1080 monitor with a 1920x1080 video with such a transition.
